
package sungardhe.workflow.ws.wsdl._1;

import jakarta.xml.ws.WebFault;

/**
 * This class was generated by the JAX-WS RI.
 * JAX-WS RI 2.2.9-b130926.1035
 * Generated source version: 2.2
 * 
 */
@WebFault(name = "DuplicateExternalEventFault", targetNamespace = "urn:sungardhe:workflow:ws:messages:1.1")
public class DuplicateExternalEventFault
        extends Exception {

    /**
     * Java type that goes as soapenv:Fault detail element.
     * 
     */
    private sungardhe.workflow.ws.messages._1.DuplicateExternalEventFault faultInfo;

    /**
     * 
     * @param faultInfo
     * @param message
     */
    public DuplicateExternalEventFault(String message,
            sungardhe.workflow.ws.messages._1.DuplicateExternalEventFault faultInfo) {
        super(message);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @param faultInfo
     * @param cause
     * @param message
     */
    public DuplicateExternalEventFault(String message,
            sungardhe.workflow.ws.messages._1.DuplicateExternalEventFault faultInfo, Throwable cause) {
        super(message, cause);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @return
     *         returns fault bean:
     *         sungardhe.workflow.ws.messages._1.DuplicateExternalEventFault
     */
    public sungardhe.workflow.ws.messages._1.DuplicateExternalEventFault getFaultInfo() {
        return faultInfo;
    }

}
